src/webportal/src/app/user/fabric/user-profile/storage-list.jsx

```diff
@@ -37,17 +37,18 @@ function normalizePath(path) {
 
 function getStorageServerUri(server) {
   let blobAddress;
+  let fileAddress;
```

Comment:
define when those variables are used and change to `const`?

Reply:
Can not define variables in `switch-case` scope. Need a new code block. Define the variables outside for convenient.

Reply:
Sorry, can not define variables in case/default clauses because of lint issue: https://eslint.org/docs/rules/no-case-declarations